packages/frontend/web-ui/src/app/store/helpers/createInitialState.spec.ts

```diff
@@ -1,12 +1,13 @@
 import { afterAll, beforeAll, describe, expect, it, jest } from '@jest/globals';
 import { createInitialState } from './createInitialState';
+import { InitialStateToken } from './models/InitialStateToken';
 import {
   STATUS_AUTH_AUTHENTICATED,
   STATUS_AUTH_NOT_AUTHENTICATED,
 } from '../data/authSliceStatus';
 
 describe(createInitialState.name, () => {
-  let getItemSpy;
+  let getItemSpy: any;
```

Comment:
Use a better type. Jest has types for this

Reply:
Done!